pkg/operator/storageclasscontroller/vmware_test.go

```diff
@@ -173,38 +198,73 @@ func TestZonalPolicyCreation(t *testing.T) {
 				t.Fatalf("error connecting to vcenter: %v", connError)
 			}
 
-			infra := tc.infra
+			for _, conn := range connections {
+				//TODO: I feel like we shouldn't create the storage policy by hand since we gotta create the FD info.
```

Comment:
Do we want to get rid of this todo? I guess it might not matter as much since its a test

Reply:
Yes.  I'll clean this up.  Thanks for catching.

Reply:
Cleaned up comments.